app/models/organisation.rb

```diff
@@ -275,6 +278,10 @@ def organisation_logo_type=(organisation_logo_type)
     self.organisation_logo_type_id = organisation_logo_type && organisation_logo_type.id
   end
 
+  def organisation_crest
+    organisation_logo_type ? organisation_logo_type.class_name : nil
```

Comment:
Would `organisation_logo_type.try(:class_name)` do the same thing here? It's also a little shorter.